Updating the Unity documentation because Microsoft haven't for the last 8 years (#6363)
microsoft/vscode-docs: docs/other/unity.md

DateApproved: 11/4/2022
MetaDescription: Visual Studio Code can replace MonoDevelop as the editor for Unity
---

> # VScode is officially unsupported

Reviewer: ```suggestion
# VS Codeis officially unsupported
```

![VScode is officially unsupported by Unity](https://forum.unity.com/threads/update-on-the-visual-studio-code-package.1302621/)
For now, the only officially supported editors are ![Visual Studio](https://visualstudio.microsoft.com/vs/unity-tools/) and ![Rider](https://www.jetbrains.com/lp/dotnet-unity/).

> # Unity Development with VS Code

Reviewer 2: hy

> # Unity Development with VS Code

Reviewer: ```suggestion
# Develop Unity C# files with VS Code
```

wouldn't this be a more suitable heading now?

> Visual Studio Code can be a great companion to Unity for editing C# files. All of the [C#](/docs/languages/csharp.md) features are supported and more.  In the screen below, you can see code colorization, bracket matching, IntelliSense, CodeLens and that's just the start.

Reviewer: ```suggestion
VScode can be a great companion to Unity for editing C# files. All of the [C#](/docs/languages/csharp.md) features are supported and more. In the screen below, you can see code colorization, bracket matching, IntelliSense, CodeLens and that's just the start.
```

Reviewer: ```suggestion
VScode can still help you edit C# files. All of the [C#](/docs/languages/csharp.md) features are supported and more. In the screen below, you can see code colorization, bracket matching, IntelliSense, CodeLens and that's just the start.
```

better?

Author: That's true, thanks for the correction
